src/summary/level-two-routes.component.tsx

```diff
@@ -41,6 +42,11 @@ export const levelTwoRoutes: PatientChartRoute[] = [
     component: MedicationsDetailedSummary,
     name: "Medications"
   },
+  {
+    url: "/patient/:patientUuid/chart/medicationsleveltwo",
```

Comment:
```suggestion
    url: "/patient/:patientUuid/chart/medications",
```

Reply:
We already have another route with that ending